textproc/mandoc/Portfile

```diff
@@ -17,12 +17,12 @@ long_description    mandoc is a suite of tools compiling mdoc, \
                     BSD manual pages, and man, the predominant \
                     historical language for UNIX manuals.
 
-master_sites        http://mdocml.bsd.lv/snapshots/
+master_sites        http://mandoc.bsd.lv/snapshots/
 
-livecheck.name	    mdocml
-distname            mdocml-${version}
-checksums           rmd160  2e2d593e3f6cfcde3bdf57b090947b355ce2b2e8 \
-                    sha256  356954f141ec6f5635e938c826f2e16e4619bb361c64d84a31f6775d030a615b
+livecheck.name      mandoc
+distname            mandoc-${version}
```

Comment:
Please note that I didn't test, but I suspect that after changing the name you probably no longer need an explicit `mandoc-${version}` (this should be the default), nor `livecheck.name`. That was needed when the portname had a different name than distfile.

Other than that it looks perfectly ok.